src/prog_models/prognostics_model.py

```diff
@@ -976,7 +976,7 @@ def check_thresholds(thresholds_met):
         save_pts = config['save_pts']
 
         # confgure optional intermediate printing
-        if config['print']:
+        if config['print']: # BOOMARK FOR MIRYAM
```

Comment:
```suggestion
        if config['print']:
```
I don't think you meant to check that in